packages/worker/src/worker-options.ts

```diff
@@ -101,7 +101,7 @@ export interface WorkerOptions {
    *
    * @experimental
    */
-  useWorkerVersioning?: boolean;
```

Comment:
This would make it inconsistent with the other SDKs. I get not wanting `worker` inside worker options, but, the redundancy isn't exactly a huge deal and this is clear in tying the feature name to the option name.

Reply:
Maybe we should change the other SDKs? It feels needlessly redundant to me in the context of worker options.